cluster/src/main/scala/io/snappydata/impl/LeadImpl.scala

```diff
@@ -183,6 +193,12 @@ class LeadImpl extends ServerImpl with Lead
 
   }
 
+  /* Returns true if LDAP Security is Enabled */
```

Comment:
As discussed earlier, The auth service is not initialized at the point where this check is placed. So auth-provider property values are considered.
But, in order to keep such code at one place, I have moved checkAuthProviderProps method to Misc class (PR: https://github.com/SnappyDataInc/snappy-store/pull/278).